src/imageResponse.js

```diff
@@ -127,12 +127,22 @@ export default {
     const clientStartTime = new Date();
     const browserImage = await image.magic(imagePath(params.name), params, response);
     browserImage.stream(params.type, (err, stdout) => {
+      if(err) {
+        response.status(500).end();
+        log('error', `Error occurred while creating live image: ${err}`);
+        return;
+      }
       const r = stdout.pipe(response);
       r.on('finish', () => {
         log('info', `Creating live image took ${new Date() - clientStartTime}ms: ${params.name}.${params.type} (${params.width}x${params.height}px, fit: ${params.fit}, blur: ${Boolean(params.blur)})`);  //eslint-disable-line max-len
         // This is to close the response while a background job will continue to process
         response.end();
       });
+      r.on('error', (error) => {
+        log('error', `The live image stream errorred with ${error}`);
+        response.end();
+        r.end();
```

Comment:
This triggers the `r.on('finish'` above as well, meaning you'll end `response` twice, while additionally producing incorrect logs about successfully creating the image.